src/like-service/app/Http/Controllers/LikeController.php

```diff
@@ -17,7 +17,9 @@ class LikeController extends BaseController
     function doLike($request)
     {
         $user_token = $request->cookie('jwt');
-        $postId = $request->header('postId');
+        $postId = $request->input('postId');
+
+        Log::notice($request->all());
```

Comment:
[DISCUSS] I think we can remove this again

Reply:
Removed